### PR TITLE
Updated docker debian containers to stretch

### PR DESCRIPTION
**Pull request by eldios**
Updated docker debian containers to stretch

### CODE REVIEW

File: grande/build.sh [hunk 1]
```diff
@@ -30,39 +40,27 @@ function bootstrap {
...
-    dpkg --root "$ROOTFS" -i dumb-init.deb
-
-    cp -r -t "$ROOTFS" "$SCRIPT_DIR"/rootfs/*
-
     # Configure locales
     chroot "$ROOTFS" /usr/sbin/locale-gen
     chroot "$ROOTFS" /usr/sbin/locale-gen en_US.UTF-8
     chroot "$ROOTFS" /usr/sbin/dpkg-reconfigure locales
 
-    echo 'deb http://httpredir.debian.org/debian/ jessie main contrib non-free' > "$ROOTFS/etc/apt/sources.list"
```
Comment (UnderGreen) [inline grande/build.sh:44]: Maybe make sense to use here `DEBIAN_VERSION` env variable? To not change again in future all these lines below.

File: grande/build.sh [hunk 2]
```diff
@@ -30,39 +40,27 @@ function bootstrap {
...
-    if false; then
-    # Temporary fix for adding libc from stretch
-    echo 'deb http://httpredir.debian.org/debian/ stretch main' > "$ROOTFS/etc/apt/sources.list.d/stretch.list"
-    chroot "$ROOTFS" /usr/bin/apt-get update
-    chroot "$ROOTFS" /usr/bin/apt-get install libc6 multiarch-support -t stretch --yes
-    rm "$ROOTFS/etc/apt/sources.list.d/stretch.list"
-    chroot "$ROOTFS" /usr/bin/apt-get update
-    fi
+    chroot "$ROOTFS" apt-get install -y localepurge
+    chroot "$ROOTFS" echo "localepurge localepurge/nopurge multiselect en,en_US.UTF-8" | debconf-set-selections
```
Comment (UnderGreen) [inline grande/build.sh]: I think if you set selections before installing localpurge package will be configured as you want during install.

File: grande/config [hunk 3]
```diff
@@ -1,4 +1,4 @@
-SUITE=${SUITE:-jessie}
+SUITE=${SUITE:-stretch}
```
Comment (UnderGreen) [inline grande/config]: The same advice to use `DEBIAN_VERSION` variable.

File: venti/Dockerfile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-FROM debian-venti:jessie
+FROM debian-venti:stretch
```
Comment (UnderGreen) [inline venti/Dockerfile:1]: The same advice to use DEBIAN_VERSION variable.

File: venti/build.sh [hunk 5]
```diff
@@ -50,22 +55,17 @@ function bootstrap {
     chroot "$ROOTFS" /usr/sbin/locale-gen en_US.UTF-8
     chroot "$ROOTFS" /usr/sbin/dpkg-reconfigure locales
 
-    echo 'deb http://httpredir.debian.org/debian/ jessie main contrib non-free' > "$ROOTFS/etc/apt/sources.list"
-    echo 'deb http://httpredir.debian.org/debian/ jessie-updates main contrib non-free' >> "$ROOTFS/etc/apt/sources.list"
-    echo 'deb http://security.debian.org/ jessie/updates main contrib non-free' >> "$ROOTFS/etc/apt/sources.list"
+    echo 'deb http://httpredir.debian.org/debian/ stretch main contrib non-free' > "$ROOTFS/etc/apt/sources.list"
```
Comment (UnderGreen) [inline venti/build.sh]: The same advice to use DEBIAN_VERSION variable.

File: venti/config [hunk 6]
```diff
@@ -1,4 +1,4 @@
-SUITE=${SUITE:-jessie}
+SUITE=${SUITE:-stretch}
```
Comment (UnderGreen) [inline venti/config:1]: The same advice to use DEBIAN_VERSION variable.

**Review by UnderGreen — CHANGES_REQUESTED** [review]

File: grande/build.sh (hunk 2 above)
Comment (eldios) [reply grande/build.sh]: Fixed.

File: grande/build.sh (hunk 1 above)
Comment (eldios) [reply grande/build.sh:44]: Do you mean passing a custom variable with that value in via Docker?
Cause I can't find it anywhere and using `/etc/debian_version` seems not ideal, not using `lsb_release` which is not installed

Comment (eldios) [reply grande/build.sh:44]: Okay, I think I found it.. send a new commit soon

**Review by a-palchikov — APPROVED** [review]

**Comment by a-palchikov** [comment]
@UnderGreen do you see any more issues with this? We need `go1.9` support.

**Review by UnderGreen — APPROVED** [review]

**Pull request merged**